crates/rpc/src/v02/types.rs

```diff
@@ -333,18 +414,82 @@ pub mod reply {
         pub nonce: TransactionNonce,
     }
 
+    #[derive(Clone, Debug, Serialize, PartialEq, Eq)]
+    #[serde(tag = "version")]
+    pub enum DeclareTransaction {
+        #[serde(rename = "0x1")]
+        V1(DeclareTransactionV1),
+        #[serde(rename = "0x2")]
+        V2(DeclareTransactionV2),
+    }
```

Comment:
I think this makes it impossible to represent a v0 declare, which we _do_ have (see https://testnet.starkscan.co/tx/0x06d346ba207eb124355960c19c737698ad37a3c920a588b741e0130ff5bd4d6d for example).